YoutubeDownloader/Views/Dialogs/AuthSetupView.xaml

```diff
@@ -67,21 +64,26 @@
                     VerticalAlignment="Center"
                     FontSize="18"
                     Text="Loading..."
-                    Visibility="{Binding IsAuthenticated, Converter={x:Static s:BoolToVisibilityConverter.InverseInstance}}" />
+                    IsVisible="{Binding !IsAuthenticated}" />
 
                 <!--  Browser  -->
-                <wpf:WebView2
+                <WebView 
+                    x:Name="WebBrowser"
+                    Loaded="WebBrowser_OnLoaded"
+                    WebViewCreated="WebBrowser_OnWebViewCreated"
+                    />
+                <!--<wpf:WebView2
                     x:Name="WebBrowser"
                     CoreWebView2InitializationCompleted="WebBrowser_OnCoreWebView2InitializationCompleted"
                     Loaded="WebBrowser_OnLoaded"
                     NavigationCompleted="WebBrowser_OnNavigationCompleted"
                     NavigationStarting="WebBrowser_OnNavigationStarting"
                     Unloaded="WebBrowser_OnUnloaded"
-                    Visibility="{Binding IsAuthenticated, Converter={x:Static s:BoolToVisibilityConverter.InverseInstance}}">
+                    IsVisible="{Binding !IsAuthenticated}">
                     <wpf:WebView2.CreationProperties>
                         <wpf:CoreWebView2CreationProperties IsInPrivateModeEnabled="True" />
                     </wpf:WebView2.CreationProperties>
-                </wpf:WebView2>
+                </wpf:WebView2>-->
```

Comment:
Hmm, I wonder if there's an x-platform web browser control on avalonia?

Reply:
Yes, there are some: https://github.com/AvaloniaCommunity/awesome-avalonia#web-browsers
But WebView.Avalonia which I used here (above the commented out code) is the only one that works on Linux, Mac & Windows, but it does not have an API for Cookies (yet). It would be the best one because it uses WebView2 and equivalents so does not add much in Size.
Alternative what I will try next is https://github.com/OutSystems/WebView which does not support Linux yet and bundles CEF (Chromium). But it currently would be the only option, so we could just disable the feature for Linux for now.

Reply:
Yeah, let's go with that. On Linux, we can just show a message box saying it's not supported.